geoapps/io/Gravity/constants.py

```diff
@@ -0,0 +1,1021 @@
+        "dependency": "forward_only",
+        "dependencyType": "hide",
+        "label": "lower bound on model",
+        "parent": "lower_bound_object",
+        "property": None,
+        "value": -1,
+    },
+    "upper_bound_object": {
+        "default": None,
+        "group": "Regularization",
+        "visible": True,
+        "dependency": "forward_only",
+        "dependencyType": "hide",
+        "label": "upper bound object",
+        "meshType": [
+            "{202C5DB1-A56D-4004-9CAD-BAAFD8899406}",
+            "{6A057FDC-B355-11E3-95BE-FD84A7FFCB88}",
+            "{F26FEBA3-ADED-494B-B9E9-B2BBCBE298E1}",
+            "{48F5054A-1C5C-4CA4-9048-80F36DC60A06}",
+        ],
+        "value": None,
+    },
+    "upper_bound": {
+        "association": "Cell",
+        "dataType": "Float",
+        "default": 1,
+        "group": "Regularization",
+        "isValue": True,
+        "visible": True,
+        "dependency": "forward_only",
+        "dependencyType": "hide",
+        "label": "upper bound on model",
+        "parent": "upper_bound_object",
+        "property": None,
+        "value": 1,
+    },
+    "parallelized": {
+        "default": True,
+        "group": "Compute",
+        "label": "use parallelization",
+        "value": True,
+    },
+    "n_cpu": {
+        "default": 1,
+        "min": 1,
+        "group": "Compute",
+        "dependency": "parallelized",
+        "dependencyType": "enabled",
+        "optional": True,
+        "enabled": False,
+        "label": "number of cpu",
+        "value": 1,
+    },
+    "max_ram": {
+        "default": 2,
+        "min": 0,
+        "group": "Compute",
+        "dependency": "parallelized",
+        "dependencyType": "enabled",
+        "optional": True,
+        "enabled": False,
+        "label": "set RAM limit",
+        "value": 2,
+    },
+    "workspace": {
+        "default": None,
+        "visible": False,
+        "enabled": False,
+        "label": "path to workspace",
+        "value": None,
+    },
+    "output_geoh5": {
+        "default": None,
+        "visible": False,
+        "enabled": False,
+        "label": "path to results geoh5py file",
+        "value": None,
+    },
+    "out_group": {
+        "default": "MVIInversion",
+        "visible": True,
+        "enabled": True,
+        "label": "results group name",
+        "value": "MVIInversion",
+    },
+    "no_data_value": {
+        "default": 0,
+        "group": "Data Options",
+        "optional": True,
+        "enabled": False,
+        "visible": False,
+        "label": "no data value",
+        "value": 0,
+    },
+    "monitoring_directory": {
+        "default": None,
+        "enabled": False,
+        "value": None,
+    },
+    "workspace_geoh5": {
+        "default": None,
+        "enabled": False,
+        "value": None,
+    },
+    "geoh5": {
+        "default": None,
+        "enabled": False,
+        "value": None,
+    },
+    "run_command": {
```

Comment:
This should be changed to "run_command": "geoapps.drivers.grav_inversion"

Reply:
yep!

Reply:
This doesn't work. The `run_command` and `conda_environment` can't be dictionaries. Just a string.